### PR TITLE
Add transifex settings

### PR DESCRIPTION
**Pull request by clonedagain**
Fixes https://github.com/OCA/reporting-engine/issues/34

### CODE REVIEW

**Comment by clonedagain** [comment]
@pedrobaeza is this how it's done?

File: .travis.yml [hunk 1]
```diff
@@ -13,9 +13,15 @@ python:
   - "2.7"
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  # Transifex settings
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: PjP88tPSwimBv4tsgn3UcQAD1heK/wcuSaSfhi2xUt/jSrOaTmWzjaW2gH/eKM1ilxPXwlPGyAIShJ2JJdBiA97hQufOeiqxdkWDctnNVVEDx2Tk0BiG3PPYyhXPgUZ+FNOnjZFF3pNWvzXTQaB0Nvz8plqp93Ov/DEyhrCxHDs=
```
Comment (pedrobaeza) [inline .travis.yml]: This should be (from the list in the instructions):

```
NUsXwVrMntcqge1ozKW+DSkP7dq+Rla6JVvFF2c89/g+zJaIqQRi8EQBLoqNwCdMk+rjpQeZt/JPELjH+EzPcmGddhDxOgVB3nUT9LvFXGCHF+NjmHXqyba4tuc7BnpG1WDD+rSlxVCt1aIjNIhhaZ4ic0rCWpKNYu/yFTsmChc=
```

Comment (clonedagain) [inline .travis.yml]: Changed, though I didn't understand how you got this hash.

**Comment by pedrobaeza** [comment]
I took it from here: https://github.com/OCA/maintainer-quality-tools/issues/194.

I have added this link in the .travis.yml sample: https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L42

**Comment by clonedagain** [comment]
Is manual setup required in Transifex to create the project and enable automatic commits?

**Comment by pedrobaeza** [comment]
No, just wait Travis ends to have the project auto-created.

**Comment by clonedagain** [comment]
ok thanks a lot.

**Pull request merged**